xds/internal/client/v2client_lds.go

```diff
@@ -68,7 +68,11 @@ func (v2c *v2Client) getRouteConfigNameFromListener(lis *xdspb.Listener) (string
 	v2c.logger.Infof("Resource with type %T, contains %v", apiLis, apiLis)
 	switch apiLis.RouteSpecifier.(type) {
 	case *httppb.HttpConnectionManager_Rds:
-		name := apiLis.GetRds().GetRouteConfigName()
+		tempRDS := apiLis.GetRds()
```

Comment:
Do we really need the `tempRDS` variable? You could simply inline that. But if you think you really want that variable, could you come up with a better name. Thanks.

Reply:
Inlined.